draft-bormann-cbor-yang-standin.md

```diff
@@ -95,9 +95,30 @@ Intermediate Encoder:
 : An encoder which isn't the original author of the data, converting it
   from legacy representation.
 
+Aggressive Intermediate Encoder:
+: An intermediate encoder that might choose to discard some
+  information of a legacy representation in order to be able to use a
+  stand-in tag.
+  Such a choice may be based on knowledge of the Decoder's handling of
+  such information (e.g, to accommodate intolerant decoders), or it
+  may be a general characteristic of the service provided by the
+  intermediate encoder (e.g., in order to serve as a legacy-eschewing
+  encoder).
+
+Legacy-Eschewing Encoder:
+: An encoder that does not generate legacy representations in places
+  where a stand-in tag might instead be used.
+  An intermediate encoder may need to be aggressive to achieve this.
+
```

Comment:
I'm a bit concerned about the negative formulation used here; slightly preferring something like this:

> An encoder that always generates stand-in tags instead of legacy representations in places where stand-in tags are allowed.

Reply:
Good point.  However, "Legacy-Eschewing Encoder" is itself a negative term; can we go fully positive?